core/src/main/java/com/intel/oap/vectorized/ArrowWritableColumnVector.java

```diff
@@ -1672,6 +1672,12 @@ final void setInt(int rowId, int value) {
       writer.setSafe(rowId, v.setScale(writer.getScale()));
     }
 
+    @Override
```

Comment:
note: since this file is also used in data source, we may need to update on arrow data source side also.

Reply:
ok, got it.